RetailSuiteAccounting/Troubleshooting/02_DisplayMissingSupplierReceipt.md

```diff
@@ -0,0 +1,62 @@
+[!!User Interface Supplier receipts](../UserInterface/01_Book.md#supplier-receipts)
+[!!Post a supplier receipt](../Operation/10_ManageReceipts.md#post-a-supplier-receipt)
+
+# Display a missing supplier receipt
+
+Check and, if necessary, modify the supplier settings to display a supplier receipt in the *Purchasing* module in order to release it.
+
+#### Error Description
+
+A supplier receipt has been entered in the system but is not displayed in the *CHECK SUPPLIER RECEIPTS* tab of the *Purchasing* module and, therefore, cannot be released.
+
+Follow the instructions below to check and modify the supplier settings in the *Accounting* module.
+
+#### Prerequisites
+
+A supplier receipt has been entered in the system, see [Acquire a supplier receipt](../Operation/10_ManageReceipts.md#acquire-a-supplier-receipt).
+
+#### Procedure
+
+*Purchasing > Check supplier receipts*
+
+  ![Overview](../../Assets/Screenshots/RetailSuiteAccounting/Book/CheckSupplierReceiptsEmpty.png "[Overview]")
+
+1. Switch to the *Overview* tab: *Accounting > Post > Button EDIT > Menu entry Supplier receipts > Tab Overview*  
+  The *Overview* tab is displayed. The supplier receipt missing in the *Purchasing* module has the **New** status (red).
+
+  ![Overview](../../Assets/Screenshots/RetailSuiteAccounting/Book/SupplierReceiptsOverview01.png "[Overview]")
+
+2. Check the supplier number for the missing supplier receipt and switch to the *Customer/supplier* tab: *Accounting > Settings > Tab DEBTORS/CREDITORS*.    
```

Comment:
Check and note?
Deutlicher machen, dass du die Supplier number für die nachfolgenden Schritte noch brauchst.